index.js

```diff
@@ -6,8 +6,10 @@ const rootURI = 'https://www.googleapis.com';
 export const refreshTokenURI = 'https://www.googleapis.com/oauth2/v4/token';
 const uploadExistingURI = id =>
     `${rootURI}/upload/chromewebstore/v1.1/items/${id}`;
-const publishURI = (id, target) =>
-    `${rootURI}/chromewebstore/v1.1/items/${id}/publish?publishTarget=${target}`;
+const publishURI = (id, target, deployPercentage) => (
+    `${rootURI}/chromewebstore/v1.1/items/${id}/publish?publishTarget=${target}`
+        + (deployPercentage == undefined ? '' : `&deployPercentage=${deployPercentage}`)
```

Comment:
Could you alter this function to build the URL via `new URL()` so you can add parameters via `url.searchParams.set()`?

Reply:
Yes, I will update my PR this way.